pyccel/codegen/printing/cwrappercode.py

```diff
@@ -273,88 +258,146 @@ def get_cast_function_call(self, cast_type, arg):
 
         return FunctionCall(cast_function, [arg])
 
+    # --------------------------------------------------------------------
+    # Functions that take care of collecting (data type, rank) checks and creating the error code
+    # --------------------------------------------------------------------
+    def _generate_TypeError_message(self, variable):
+        """
+        Generate TypeError message from the variable informations (datatype, precision)
+        """
+        dtype     = variable.dtype
+        precision = '' if isinstance(dtype, NativeBool) else '{} bit '.format(variable.precision * 2 * 8)\
+                       if isinstance(dtype, NativeComplex) else '{} bit '.format(variable.precision * 8)
```

Comment:
I don't find this very readable in one line. Is there a reason you didn't want to create a block?

Reply:
Actually, no reason, just bad habit i will change it

Reply:
done 42ce0db